migtests/tests/analyze-schema/expected_issues.json

```diff
@@ -20,6 +20,26 @@
             "GH": "https://github.com/yugabyte/yb-voyager/issues/1542",
```

Comment:
Let's add assessment end-to-end tests as well? You can skip the JSON_TABLE

Reply:
I was planning to but these features in this PR are PG17 features so we need pg_dump 17 as well to get have these end to end tests

Reply:
Added Agg function test but not running it in end-to-end tests as can't upgrade to PG16 in misc-migtests.yml easily as there are some validation failures because of different ways of dumping the VIEW/MVIEW's select stmt from PG15 -> PG16. 
This has to be separately by fixing the expected files. So will enable these once we upgrade